tests/unit/test_network_connections.py

```diff
@@ -143,6 +143,7 @@ def setUp(self):
                 "gateway4": None,
                 "route_metric4": None,
                 "auto6": True,
+                "ipv6_disabled": False,
```

Comment:
`auto6:True` and `ipv6_disabled:False` sounds like an invalid combination.

Reply:
the argvalidators are supposed to either validate (and reject such a setting) or to normalize it -- whatever is more sensible.

Other parts of the code then can rely that parameters have sensible values (also with respect to each other).

Reply:
I deleted or changed into `"ipv6_disabled": None,`, `tox -e py37` tests failed.
Maybe thomas is right, we should validate the value in argvalidators first.

Reply:
> auto6:True and ipv6_disabled:False sounds like an invalid combination.

I think Till meant "auto6:True and ipv6_disabled:**True** sounds like an invalid combination."

Anyway, in the latest version of the code this is fixed and rejected as invalid.